internal/api/utils.go

```diff
@@ -44,3 +46,12 @@ func unwrapOrDefault[T any](value *T, defaultValue T) T {
 	}
 	return *value
 }
+
+func GenerateToken() string {
+	tokenBytes := make([]byte, 12)
```

Comment:
```suggestion
	tokenBytes := make([]byte, 24)
```
maybe 12 is a bit short